src/results.md

```diff
@@ -4,34 +4,49 @@ title: Results
 permalink: /results/
 include: true
 ---
-Having the world’s largest database of Syriac letter forms allows one to both describe and quantify important developments in Syriac manuscript culture. In some cases, this results in computer assisted paleography. That is, automated results provide data that help human paleographers better localize Syriac manuscripts. In other cases, our data challenges long held assumptions in the field (e.g. the traditional way of categorizing Syriac script) or opens up new fields of inquiry (e.g. the relationship between script and genre). Although more details appear in project publications, the following provide several quick case examples for how large data sets, digital analysis, and visual analytics can help researchers address key philological and historical problems.
+Having the world’s largest database of Syriac letter forms allows one to both describe and quantify important developments in Syriac manuscript culture. In some cases, this helps with computer assisted paleography. That is, automated results provide data that allow human paleographers to better localize Syriac manuscripts. In other cases, project data challenges long held assumptions in the field (e.g. the traditional way of categorizing Syriac script) or opens up new fields of inquiry (e.g. the relationship between script and genre). Although more details appear in project publications, the following provide several quick case examples for how large data sets, digital analysis, and visual analytics can help researchers address key philological and historical problems. 
```

Comment:
e.g. -> e.g., [two occurrences]

Reply:
Can you clarify what you mean here?

Reply:
I think the general preference on this side of the Atlantic is to provide a comma after "e.g.", as is done with "i.e.", but it's not a universal consensus, so given the transatlantic nature of our team (and this project) I won't insist...

Reply:
Ah. I'm fine with adding it if you'd like (though I see you just merged). The formatting of your comment was confusing to me, and I associated the comma with breaking off your own bracketed two occurrences.